src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -85,6 +85,19 @@ namespace DeviceLayer {
 
 ConnectivityManagerImpl ConnectivityManagerImpl::sInstance;
 
+void ConnectivityManagerImpl::UpdateEthernetNetworkingStatus()
+{
+    if (mpStatusChangeCallback != nullptr)
+    {
+        if (mEthIfName[0] != '\0')
```

Comment:
This could just use `GetEthernetIfName()` and `CharSpan::fromString` instead of doing strnlen "by hand"